core/src/main/java/org/elasticsearch/cli/ExitCodes.java

```diff
@@ -37,6 +37,7 @@
     public static final int PROTOCOL = 76;       /* remote error in protocol */
     public static final int NOPERM = 77;         /* permission denied */
     public static final int CONFIG = 78;         /* configuration error */
+    public static final int PLUGIN_EXISTS = 79;  /* config error; plugin already exists */
```

Comment:
I thought it would be prudent to add as it is a subclass of configuration errors; however I can see the value in your comment as future exit codes should break down the class of errors for it to be handled by scripts that consume the plugins tool.